.github/actions/setup-kurtosis-type1/action.yml

```diff
@@ -36,19 +64,46 @@ runs:
       working-directory: ./kurtosis-cdk
       shell: bash
       run: |
+        sed -i '/zkevm.sequencer-batch-seal-time:/d' templates/cdk-erigon/config.yml
+        sed -i '/zkevm.sequencer-non-empty-batch-seal-time:/d' templates/cdk-erigon/config.yml
+        sed -i '/zkevm\.sequencer-initial-fork-id/d' ./templates/cdk-erigon/config.yml
+        sed -i '/sentry.drop-useless-peers:/d' templates/cdk-erigon/config.yml
+        sed -i '/zkevm\.pool-manager-url/d' ./templates/cdk-erigon/config.yml
         sed -i '/zkevm.datastream-version:/d' ./templates/cdk-erigon/config.yml
+        sed -i 's/zkevm\.disable-virtual-counters: false/zkevm.disable-virtual-counters: true/' ./templates/cdk-erigon/config.yml
         sed -i '$a\zkevm.honour-chainspec: true' ./templates/cdk-erigon/config.yml
         sed -i '$a\zkevm.initial-commitment: pmt' ./templates/cdk-erigon/config.yml
+        sed -i 's/"londonBlock": [0-9]\+/"londonBlock": 0/' ./templates/cdk-erigon/chainspec.json
```

Comment:
these blocks are set in the templates and depend on enable_normalcy which as passed in "Deploy Kurtosis CDK package" step. I see you removed passing it, and added more sed commands here. 
I'd just like to agree on some approach because I removed these in my commit. you are readding them so we just need to align on this.

Reply:
ok, fine with me. I normally tend to avoid code dups and since it is already handled in the kurtosis cdk templates and hope it is not likely no change, just more lines added there for pectra and other hardforks, we wont have to change it here. But no strong opinion here.